packages/components/src/Text/types.ts

```diff
@@ -0,0 +1,188 @@
+import { Props as TruncateProps } from '../Truncate/useTruncate';
```

Comment:
@ItsJonQ This is what I was talking about using `.ts` files to define the types that we then import into the JSDoc 😉

Reply:
Thank goodness for that!~!!